examples/penv.yaml

```
@@ -31,18 +23,23 @@ functions:
-    - |
+    help: Activate a venv
+    args:
+      - name: str!
+      - --interpreter | -i: str = "python"
+    run: |
       if venv_exists(name):
           print(f"~ sourcing {name}")
       else:
           print(f"~ creating {name}")
           $ f{interpreter} -m venv f{os.path.join(DEFAULT_VENV_STORE, name)}
       
```

yamllint (1.35.1)

[error] trailing spaces

(trailing-spaces)